extensions/amazon-aws/src/glue-jobs.tsx

```diff
@@ -0,0 +1,56 @@
+import { GlueClient, ListJobsCommand } from "@aws-sdk/client-glue";
+import { Action, ActionPanel, Icon, List } from "@raycast/api";
+import { useCachedPromise } from "@raycast/utils";
+import AWSProfileDropdown from "./components/searchbar/aws-profile-dropdown";
+import { isReadyToFetch, resourceToConsoleLink } from "./util";
+import { AwsAction } from "./components/common/action";
+
+export default function GlueJobs() {
+  const { data: glueJobs, error, isLoading, revalidate } = useCachedPromise(fetchJobs);
```

Comment:
suggestion (non-blocking): we should use `mutate` instead of `revalidate` that we get from `useCachedPromise`. That way we can exercise more control further down the line.